lib/event_date.php

```diff
@@ -101,11 +102,11 @@ public function getDescriptionAsPlaintext() : ?string
     {
         return strip_tags(html_entity_decode($this->description));
     }
-    public function getIcsStatus()
+    public function getIcsStatus(): int
     {
         return strip_tags($this->eventStatus);
     }
```

Comment:
```suggestion
    public function getIcsStatus(): string
    {
        return strip_tags($this->eventStatus);
    }
```

Da weiß ich, dass ein String rauskommt, weil ICS / Structured Data einen Text-Status haben